frontend/public/module/k8s/k8s-actions.ts

```diff
@@ -67,6 +72,39 @@ const actions = {
       });
   },
 
+  setPrinterColumns: (model, name) => dispatch => {
+    const kind = model.crd ? CustomResourceDefinitionModel : model;
+    k8sGet(kind, name).then(obj => {
```

Comment:
> Not all users can get CRDs. I'm not sure this approach will work.

So they will be just able to list them ?

Reply:
No, it looks like normal users can't list them either.

```

❯ oc --as=user@example.com get crds
No resources found.
Error from server (Forbidden): customresourcedefinitions.apiextensions.k8s.io is forbidden: User "user@example.com" cannot list customresourcedefinitions.apiextensions.k8s.io at the cluster scope: no RBAC policy matched

❯ oc adm policy who-can list crds
[...]
Groups: system:cluster-admins
        system:cluster-readers
        system:masters
```

Reply:
so even using the Table accept header would not solve this, since its a List call and would be denied as well.
Having a link to a specific CRD in a nav section makes sense now, since the section link would be seen only to users that will be able to list that particulat CRD. right ?

Reply:
@jhadvig you can have authority to list CR instances without being able to get the CRD itself

Reply:
@spadgett so other solution would be to try to get the CRD itself, and if that fails we could print a plain table as is with disabled sorting. if on the other hand user will be able to get the CRD we could use it to assemble the table.

Other approach that I've already tried but came to be a bit more hackish is to create our own list of resources from the table returned [table representation](https://gist.github.com/jhadvig/2d5059d9f282a1bc47fc176fc2e747ba), by converting into an array of [flattened objects](https://gist.github.com/jhadvig/5513ede2fa19fb89c49a496a079cfe9e), with which List component "could" work, although some assumptions would need to be done (eg. the [names](https://gist.github.com/jhadvig/2d5059d9f282a1bc47fc176fc2e747ba#file-table-representation-json-L10) are  not unique, ...). 
Not sure if transforming the data is the right approach but while was digging into it I found more and more problems that we would need take care of...